programs/scite.json

```
@@ -0,0 +1,15 @@
+{
+    "name": "SciTE",
+    "files": [
+        {
+            "path": "$HOME/.SciTE.session",
+            "movable": true,
+            "help": "Export the following environment variables:\n\n```bash\nexport SciTE_USERHOME=\"$XDG_CONFIG_HOME\"/SciTE\n```\n",
```

```suggestion
            "help": "Export the following environment variables:\n\n```bash\nexport SciTE_USERHOME=\"$XDG_CONFIG_HOME\"/SciTE\n```\n"
```